project/plugins.sbt

```diff
@@ -1,5 +1,5 @@
-addSbtPlugin("com.scalawilliam.esbeetee" % "sbt-vspp" % "0.4.11")
+//addSbtPlugin("com.scalawilliam.esbeetee" % "sbt-vspp" % "0.4.11")
```

Comment:
If there was an SBT 1.2.8-compatible version, I wouldn't be doing this in the first place :)

You'll need to publish an 1.2.8-compatible version and re-enable that dependency.